spec/system/signup/index.spec.rb

```diff
@@ -0,0 +1,127 @@
+require 'rails_helper'
+
+RSpec.describe 'Signup feature' do
+	it 'should create new user successfully' do
+		visit new_user_registration_path
+		expect(page).to have_content('Sign up new account')
+
+		fill_in('user_full_name', with: 'John Doe')
+		select "ASHA", :from => "user_role"
+		fill_in('user_email', with: 'johndoe@example.com')
+		fill_in('user_phone', with: '9988776677')
+		fill_in('user_password', with: 'JohnDoe')
+
+		click_button('Sign Up', exact: true)
+		expect(current_path).to eql('/')
+
+		expect(page).to have_content('Your account is not verified!')
+	end
+
```

Comment:
Indentation seems too much. 
Does prettier support ruby file 🤔 
Otherwise we have to install rufo extension 😅

Reply:
@peeyush14goyal What do you think?